src/pages/Checklist/ChecklistPage.tsx

```diff
@@ -77,105 +75,15 @@ const ChecklistPage = (): JSX.Element => {
-                        details.checkList.mcPkgNo,
-                        details.checkList.formularType,
-                    ].filter((x) => x != null)}
-                    attachments={details.checkList.attachmentCount}
-                />
-            );
-        }
-        if (fetchDetailsStatus === AsyncStatus.ERROR) {
-            return (
-                <DetailsWrapper>
-                    Unable to load details. Please reload
-                </DetailsWrapper>
-            );
-        }
-        return (
-            <DetailsWrapper>
-                <DotProgress color="primary" />
-            </DetailsWrapper>
-        );
-    };
-
-    const determineFooterToRender = (): JSX.Element => {
-        if (
-            (fetchPunchListStatus === AsyncStatus.SUCCESS ||
-                fetchPunchListStatus === AsyncStatus.EMPTY_RESPONSE) &&
-            punchList != undefined
-        ) {
-            return (
-                <NavigationFooter>
-                    <FooterButton
-                        active={
-                            !history.location.pathname.includes(
-                                '/punch-list'
-                            ) &&
-                            !history.location.pathname.includes('/tag-info')
-                        }
-                        goTo={(): void => history.push(`${url}`)}
-                        icon={<EdsIcon name="playlist_added" />}
-                        label={'Checklist'}
-                    />
-                    <FooterButton
-                        active={history.location.pathname.includes('/tag-info')}
-                        goTo={(): void => history.push(`${url}/tag-info`)}
-                        icon={<EdsIcon name="tag" />}
-                        label={'Tag info'}
-                    />
-                    <FooterButton
-                        active={history.location.pathname.includes(
-                            '/punch-list'
-                        )}
-                        goTo={(): void => history.push(`${url}/punch-list`)}
-                        icon={<EdsIcon name="warning_outlined" />}
-                        label={'Punch list'}
-                        numberOfItems={punchList.length}
-                    />
-                </NavigationFooter>
-            );
-        }
-        if (fetchPunchListStatus === AsyncStatus.ERROR) {
-            return (
-                <NavigationFooterShell>
-                    <p>Unable to load footer. Please reload</p>
-                </NavigationFooterShell>
-            );
-        }
-        return (
-            <NavigationFooterShell>
-                <DotProgress color="primary" />
-            </NavigationFooterShell>
-        );
-    };
-
     return (
-        <>
+        <main>
             <Navbar
                 leftContent={
                     <BackButton
                         to={
                             history.location.pathname.includes('/new-punch')
```

Comment:
Skill ut logikken slik at denne blir enklere å lese, for eksempel slik:
to={ isOnNewPunchPage ? goBackOnce() : goBackTwice() }